cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastTimeSeries.java

```diff
@@ -0,0 +1,4 @@
+package cwms.cda.data.dto.forecast;
+

```

Comment:
I suspect this will likely be better named "ForecastTimeSeriesList" but the roughout here looks reasonable.